### DISCUSSION TITLE
exponential, cubic, circular

### DISCUSSION THREAD

**Discussion by Electricavisuals** [mrachinskiy/commotion#31] [Ideas]
Hi, Next Step, should be to make it offset keys in time as exponentially , cubically, or circular curved , apart of linear...
It's already in KeyTransfer and ATK for 3dsmax
thanks!

**Comment by mrachinskiy** [comment]
Hi, I need an example showing how it works.

**Reply by Electricavisuals** [reply]
https://youtu.be/d5hXTHAOP94?t=76
the video is at the point of what I mean, but maybe you need to see it from the beginning...
thanks for the fast answer!
